src/main/java/frc/robot/subsystems/vision/Limelight.java

```diff
@@ -0,0 +1,127 @@
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.net.PortForwarder;
+import edu.wpi.first.networktables.*;
+import frc.robot.subsystems.LoggedSubsystem;
+
+import java.util.Optional;
+import java.util.OptionalDouble;
+
+public class Limelight extends LoggedSubsystem<LimelightLogInputs> {
+    public static Limelight INSTANCE = null;
+
+    private final NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+    private final DoubleSubscriber tx = table.getDoubleTopic("tx").subscribe(0.0);
+    private final DoubleSubscriber ty = table.getDoubleTopic("ty").subscribe(0.0);
+    private final IntegerSubscriber tv = table.getIntegerTopic("tv").subscribe(0);
+    private final DoubleSubscriber ts = table.getDoubleTopic("ts").subscribe(0.0);
+    private final IntegerSubscriber tid = table.getIntegerTopic("tid").subscribe(0);
+
+    private Limelight() {
+        super(new LimelightLogInputs());
+        PortForwarder.add(5800, "limelight.local", 5800);
+        PortForwarder.add(5801, "limelight.local", 5801);
+        PortForwarder.add(5802, "limelight.local", 5802);
+        PortForwarder.add(5803, "limelight.local", 5803);
+        PortForwarder.add(5804, "limelight.local", 5804);
+        PortForwarder.add(5805, "limelight.local", 5805);
```

Comment:
There is a lot of repetition in the code block. Find another way to reduce repetition.